.github/workflows/tests_linux.yml

```diff
@@ -109,7 +109,7 @@ jobs:
       - name: Install lightning.qubit device
         run: |
           cd main
-          python setup.py build_ext -i --define="ENABLE_KOKKOS=OFF;ENABLE_PYTHON=ON;CMAKE_CXX_COMPILER=$(which g++-$GCC_VERSION)"
+          python setup.py build_ext -i --define="PYTHON_EXECUTABLE=$(which python);ENABLE_KOKKOS=OFF;ENABLE_PYTHON=ON;CMAKE_CXX_COMPILER=$(which g++-$GCC_VERSION)"
```

Comment:
Do we need the `PYTHON_EXECUTABLE` here and below? I'd assume since the calling python will be on path, it should find it first.

Reply:
I'm not sure about this, but it used to be forced in `setup.py`. I can remove it and see what happens.

Reply:
You were right, not required.